plugins/woocommerce/client/blocks/assets/js/blocks/accordion/accordion-group/frontend.js

```diff
@@ -3,36 +3,40 @@
  */
 import { store, getContext } from '@wordpress/interactivity';
 
-const { state } = store( 'woocommerce/accordion', {
-	state: {
-		get isOpen() {
-			const { isOpen, id } = getContext();
-			return isOpen.includes( id );
+const { state } = store(
+	'woocommerce/accordion',
+	{
+		state: {
+			get isOpen() {
+				const { isOpen, id } = getContext();
+				return isOpen.includes( id );
+			},
 		},
-	},
-	actions: {
-		toggle: () => {
-			const context = getContext();
-			const { id, autoclose } = context;
+		actions: {
+			toggle: () => {
+				const context = getContext();
+				const { id, autoclose } = context;
 
-			if ( autoclose ) {
-				context.isOpen = state.isOpen ? [] : [ id ];
-			} else if ( state.isOpen ) {
-				context.isOpen = context.isOpen.filter(
-					( item ) => item !== id
-				);
-			} else {
-				context.isOpen.push( id );
-			}
+				if ( autoclose ) {
+					context.isOpen = state.isOpen ? [] : [ id ];
+				} else if ( state.isOpen ) {
+					context.isOpen = context.isOpen.filter(
+						( item ) => item !== id
+					);
+				} else {
+					context.isOpen.push( id );
+				}
+			},
 		},
-	},
-	callbacks: {
-		initIsOpen: () => {
-			const context = getContext();
-			const { id, openByDefault } = context;
-			if ( openByDefault ) {
-				context.isOpen.push( id );
-			}
+		callbacks: {
+			initIsOpen: () => {
+				const context = getContext();
+				const { id, openByDefault } = context;
+				if ( openByDefault ) {
+					context.isOpen.push( id );
+				}
+			},
 		},
 	},
-} );
+	{ lock: true }
```

Comment:
I wonder why we lock this and `plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/frontend.ts` completely but allow unlocking for others?

Reply:
@dinhtungdu I believe it's because we want to allow experimentation with add to cart with options available actions. such as `setQuantity`, it may become a public API but we don't want to promise its stable.

Where we've said `lock: true` we don't think there will be a value in exposing the API at all.

Reply:
> but allow unlocking for others?

Sorry, but what do you mean by this? All the stores should be locked for now.

Reply:
@luisherranz I mean, we can still unlock stores that are locked with a string, but for ones locked with a `boolean`, we can't. [Ref](https://developer.wordpress.org/block-editor/reference-guides/interactivity-api/api-reference/#private-stores).

Reply:
The stores locked with `true` can be unlocked using [the following string](https://github.com/woocommerce/woocommerce/blob/d0e2c0bc7badced522e6b0f4d9d271808b45fd38/plugins/woocommerce/client/blocks/assets/js/base/stores/woocommerce/cart.ts#L185): `I acknowledge that using a private store means my plugin will inevitably break on the next store release.`

It is the same strategy that Core is using [for the editor packages](https://github.com/WordPress/gutenberg/blob/7506dbbe49367f0641d339473b4416f579bc9376/packages/block-editor/src/lock-unlock.ts).